database/mongodb.go

```diff
@@ -240,3 +240,14 @@ func (m *MongoDB) GetMessages() ([]model.AdminMessage, error) {
 	err := m.messages.Find(bson.M{}).All(&messages)
 	return messages, err
 }
+
+// UserExists tests if a given user exists in the admin database
+func (m *MongoDB) UserExists(uname string) bool {
```

Comment:
I think `uname` should be `username`.

Also, we should definitely allow this method to return an error.